doc/visual-programming/source/widgets/unsupervised/distancetransformation.rst

```diff
@@ -0,0 +1,57 @@
+Distance Transformation
+=======================
+
+.. figure:: icons/distance-transformation.png
+
+Transforms distances in a data set. 
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Distances**
+
+A distance matrix
+
+**Outputs**:
+
+-  **Distances**
+
+A distance matrix
+
+Description
+-----------
+
+The **Distances Transformation** widget is used for the normalization and inversion of distance matrices. The normalization of data is necessary to bring all the variables into proportion with one another. 
+
+.. figure:: images/DistanceTransformation-stamped.png 
+
+1. Choose the type of `Normalization <https://en.wikipedia.org/wiki/Database_normalization>`_:
+
+   - **No normalization**
+   - **To interval [0, 1]**
+   - **To interval [-1, 1]**
+   - `Sigmoid function <https://en.wikipedia.org/wiki/Sigmoid_function>`_: 1/(1+exp(-X)) 
+
+2. Choose the type of Inversion: 
```

Comment:
This is strangely undocumented in code and I'm not sure why it's called 'Inversion'. I wish we could explain it somehow better. Ideas?